index.js

```diff
@@ -8,19 +8,27 @@ const fetch = require('./fetch')
 const debug = createDebug('iheart')
 
 // go through a CORS reverse proxy so that these APIs work in the web browser
-const corsProxy = 'https://cors.now.sh/'
+const corsDefaultProxy = 'https://cors.now.sh/'
```

Comment:
Let's make the default be empty string when in Node.js

```js
const corsDefaultProxy = process.browser ? 'https://cors.now.sh/' : '' 
```

Reply:
Roger that. Never stop learning